examples/startup_technical_ideas/app.py

```diff
@@ -0,0 +1,44 @@
+import agenta as ag
+from agenta import FloatParam, MessagesInput, MultipleChoiceParam
+from openai import OpenAI
+
+client = OpenAI()
+
+SYSTEM_PROMPT = "You have expertise in offering technical ideas to startups."
+CHAT_LLM_GPT = [
+    "gpt-3.5-turbo-16k",
+    "gpt-3.5-turbo-0301",
+    "gpt-3.5-turbo-0613",
+    "gpt-3.5-turbo-16k-0613",
+    "gpt-4",
+]
+
+ag.init(app_name="technical_ideas", base_name="app")
+ag.config.default(
+    temperature=FloatParam(0.2),
+    model=MultipleChoiceParam("gpt-3.5-turbo", CHAT_LLM_GPT),
+    max_tokens=ag.IntParam(-1, -1, 4000),
+)
+
+
+@ag.entrypoint
+def chat(
+    inputs: MessagesInput = MessagesInput(
+        [{"role": "system", "content": SYSTEM_PROMPT}]
```

Comment:
I would not have the system prompt in the default input but as part of the configuration!
i.e. you need to a append 

to do later
```python
messages = {"role": "system", "content": ag.config.system_prompt} + messages
```